.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!--
+Please only file issues here that you believe represent actual bugs or feature requests for the Stripe Terminal iOS SDK.
+
+If you're having general trouble with your Stripe integration, please reach out to support using the form at https://support.stripe.com/ (preferred) or via email to support@stripe.com.
```

Comment:
We might want to change this to the Terminal specific support email, `support-terminal@stripe.com`